### ISSUE TITLE
output DRR quality and training loss

### ISSUE THREAD

**Issue by gretareitenbach** [eigenvivek/xvr#42]
I am having issues with stagnant training loss and low-quality output DRRs. Here is a link to my agnostic, finetuned, and registration code. I have also attached images of the wandb training log and input/output images. Thank you!

https://gist.github.com/gretareitenbach/0f7770df0d2a666230927ca54fdb602f
